app/_src/policies/traffic-metrics.md

```diff
@@ -54,7 +54,8 @@ spec:
     - name: prometheus-1
       type: prometheus
       conf:
-        skipMTLS: false
```

Comment:
We should fix the example in line 102 for universal which configures tls disabled but here we create tls enabled.